eth_tester/backends/pyevm/main.py

```diff
@@ -128,16 +129,24 @@ def generate_genesis_state_for_keys(account_keys, overrides=None):
 
 
 def get_default_genesis_params(overrides=None):
-    # commented out params became un-configurable in py-evm during London refactor
+    # Commented out params became un-configurable in py-evm during London refactor.
+    # Post-merge now. Set genesis params to expect post-merge validation. If PoS field
+    # value defaults are not desired, use the overrides option.
+    from eth.constants import (
```

Comment:
I'm curious why you put these here instead of at the top of the file?

Reply:
I went back and forth on it a bit but I think up top may be better. We do have to play the game of `is_supported_pyevm_version_available()` else `None`. I think that's ok since these methods should never really be called if py-evm isn't available. Thoughts?

Reply:
Yeah, I think up top might be a little clearer, but not a strong preference